### ISSUE TITLE
False positive for document.fonts not supported in Edge 79

### ISSUE THREAD

**Issue by Bogidon** [amilajack/eslint-plugin-compat#292]
Not sure if this is the right place to report the issue (as opposed to in caniuse-db), but I'm getting the error:
```
document.fonts() is not supported in Edge 79
```

When Edge 79 does support it: https://developer.microsoft.com/en-us/microsoft-edge/status/cssfontloadingapi/?q=edgeChromium%3AShipped%20edge%3ADeprecated%20edge%3A%27Not%20Supported%27
